index.html

```diff
@@ -223,22 +266,43 @@ <h3 class="skills-header">Skills</h3>
               <li>Terminal</li>
             </ul>
           </section>
+          <div class="package-background"></div>
+        </article>
+        <section class="about-container">
+          <h2 class="about-header">About</h2>
+          <p class="about-paragraph">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ea eiusmod
+          </p>
+          <button class="cv-button">
+            Get my cv
+            <span id="cv-span">
+              <img
+                src="./assests/images/ic_download.svg"
+                alt="get my cv icon"
+              />
+            </span>
+          </button>
         </section>
       </section>
       <!--the third part of the project-->
       <section class="contact">
-        <h3 class="contact-header">Get started</h3>
-        <p class="contact-paragraph">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna.
-        </p>
+        <article class="introduction-contact">
+          <h3 class="contact-header">Get started</h3>
+          <p class="contact-paragraph">
```

Comment:
- [ ] Please make sure this section is in the center as on the design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/160180767-c8753ab2-a102-4765-a8ed-e87d2919bd2c.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/160180820-1bfc8780-9eda-475e-8400-9b44486834b6.png)